specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/compute.json

```diff
@@ -9198,6 +9204,12 @@
           "$ref": "#/definitions/CapacityReservationInstanceView",
           "readOnly": true,
           "description": "The Capacity reservation instance view."
+        },
+        "timeCreated": {
```

Comment:
If you're adding a new property, you will need to create a new [Example ](https://github.com/Azure/azure-rest-api-specs/tree/feature/cplat-2021-11-01/specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/examples) file or edit an existing one for an API operation using this property.

Reply:
Edited the base examples for CR, DH, VM, and VMSS